src/index.tsx

```diff
@@ -1,19 +1,35 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-
 import * as Sentry from '@sentry/react';
 import { BrowserTracing } from '@sentry/tracing';
-import { CaptureConsole as CaptureConsoleIntegration } from '@sentry/integrations';
 
 import App from './App';
 
 Sentry.init({
   dsn: process.env.SENTRY_DSN,
   integrations: [
-    new BrowserTracing(),
-    new CaptureConsoleIntegration({ levels: ['error'] }),
+    new BrowserTracing()
   ],
   tracesSampleRate: 1.0,
+  beforeSend(event, hint) {
+    const error = hint.originalException;
+
+    const ignoreErrors = [
+      /GraphQL/i, // filter out graphql errors
+      /Failed to fetch/i // filter out failed to fetch network errors
+    ]
+
+    // filter out blacklisted errors
+    if (error && error.message) {
+      for (const filter of ignoreErrors) {
+        if (error.message.match(filter)) {
```

Comment:
I don't think your matching logic is required if you define ignoreErrors outside of the function as an initialization param: https://docs.sentry.io/clients/javascript/config/

Reply:
Good point. I've seen that not sure why I added as a function. Maybe I was thinking we will need more than regex. Will change to that.